.github/workflows/ci.yaml

```diff
@@ -91,7 +91,11 @@ jobs:
     # Run unit tests
     - name: Run unit tests
       run: |
-        dotnet test --logger trx LoRaEngine/test/LoRaWanNetworkServer.Test/*.csproj -r LoRaEngine/test/TestResults/  &&  dotnet test --logger trx LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaToolsTest/*.csproj -r LoRaEngine/test/TestResults/ && dotnet test --logger trx LoRaEngine/test/LoraKeysManagerFacade.Test/*.csproj -r LoRaEngine/test/TestResults/
+        dotnet test --configuration ${{ env.buildConfiguration }} --logger trx -r \
+          LoRaEngine/test/TestResults \
+          /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura \
+          --filter 'FullyQualifiedName~LoRaWanTest|FullyQualifiedName~LoRaWan.NetworkServer.Test|FullyQualifiedName~LoraKeysManagerFacade.Test' \
```

Comment:
I was looking for the tools tests, until I realized, they are the LoRaWanTest.

Reply:
Yeah, there was quite a bit of confusion for me there as well. It seems some of the assembly names, project names and namespaces are not aligned. It would be good to clean that up eventually. I am [taking notes](https://github.com/Azure/iotedge-lorawan-starterkit/projects/8#card-69834110).